content/en/docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -65,12 +65,22 @@ The output shows the text written by the postStart handler:
 
 ## Discussion
 
+Unlike with Jobs or `initContainers` the handlers for postStart and preStop will
```

Comment:
Not sure how the Job resource is relevant here; @Anticom could you clarify?

Maybe:
```suggestion
For a Container, its handlers for `postStart` and `preStop`
run inside that Container. If the container's filesystem is writeable, or
there are writeable volumes, `postStart` and `preStop` handles can modify
those.

This is different from an init container, which can modify its own filesystem
(plus mounted volumes), and cannot modify the overall filesystem of any
other containers.
```
?

Reply:
@sftim To my knowledge there are only three ways to perform "one-off" program executions:
1. Jobs
1. initContainers
1. lifecycle event hooks

By one-off executions I mean non-daemonized processes that are supposed to end at some time and their exit code signaling the outcome. However all three possibilities serve different purposes and hence behave slightly differently. I've mentioned Jobs and initContainers because I wanted to clarify, that you can't use a Job nor an initContainer to modify the container-local fs, however with lifecycle event hooks you can.